p/paping/paping_rhel_8.4.sh

```diff
@@ -0,0 +1,27 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : paping
+# Version       : v1.5.5
+# Source repo   : https://github.com/koolhazz/paping.git
+# Tested on     : RHEL 8.4
+# Language      : C++
+# Script License: Apache License Version 2.0
+# Maintainer    : sachin.kakatkar@ibm.com
```

Comment:
Updated the header as per the latest guidelines.
Missing "Travis-check" field in the header.